image/vulnerabilities/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8-minimal
+ARG BASE_TAG=latest
+
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+ARG LABEL_VERSION
+ARG LABEL_RELEASE
+ARG QUAY_TAG_EXPIRATION
+
+LABEL name="definitions" \
+      vendor="StackRox" \
+      maintainer="https://stackrox.io/" \
+      summary="Vulnerability definitions for the StackRox Security Platform" \
+      description="This image contains vulnerability definitions used by StackRox Scanner V4." \
+      version="${LABEL_VERSION}" \
+      release="${LABEL_RELEASE}" \
+      quay.expires-after="${QUAY_TAG_EXPIRATION}"
+
+COPY scanner-vuln-updates.zip /app/scanner-vuln-updates.zip
```

Comment:
nit: Should we just have it in the root directory? Not sure if `app` makes sense. Or perhaps there's some Linux standard for putting random data files? Not sure if anyone else has opinions on this

Reply:
> Or perhaps there's some Linux standard for putting random data files?

`/srv/` is the closest to what we are doing here. Maybe `/opt/` would also make sense. Although, in containers, FHS loses its relevance a bit so any directory the images' base OS won't bother is fine.

Reply:
Cool. Yeah, I don't have a strong opinion